wrappers/python/nntile/utils/constructors.py

```diff
@@ -0,0 +1,55 @@
+# @copyright (c) 2022-present Skolkovo Institute of Science and Technology
+#                              (Skoltech), Russia. All rights reserved.
+#                2023-present Artificial Intelligence Research Institute
+#                              (AIRI), Russia. All rights reserved.
+#
+# NNTile is software framework for fast training of big neural networks on
+# distributed-memory heterogeneous systems based on StarPU runtime system.
+#
+# @file wrappers/python/nntile/pipeline.py
+# TRaining pipeline of NNTile Python package
+#
+# @version 1.0.0
+
+from nntile.nntile_core.tensor import TensorTraits, Tensor_fp32, Tensor_fp64, \
+        Tensor_int64, Tensor_fp16, Tensor_bool
+from nntile.functions import fill_async
+import numpy as np
+
+nnt2np_type_mapping = {
+    Tensor_fp32: np.float32,
+    Tensor_fp64: np.float64
+}
+
+np2nnt_type_mapping = {
+    np.float32: Tensor_fp32,
+    np.float64: Tensor_fp64
+}
+
+
+
+def tensor_from_numpy(A: np.array, basetile_shape: tuple = None, mpi_distr = [0], next_tag = 0):
+    A_traits = TensorTraits(A.shape, basetile_shape if basetile_shape else A.shape)
+    
+    A_value = np2nnt_type_mapping[A.type](A_traits, mpi_distr, next_tag)
+
+    A_value.from_array(A)   
+    return A_value
+
+
+def tensor_to_numpy(tensor_nnt):
+    np_res = np.zeros(tensor_nnt.shape, order='F').astype(np2nnt_type_mapping[type(tensor_nnt)])
+    tensor_nnt.to_array(np_res)
+    return np_res
+
+def zeros(shape, dtype=Tensor_fp32):
+    np_dtype = nnt2np_type_mapping[dtype]
+    return tensor_from_numpy(np.zeros(shape).astype(np_dtype))
```

Comment:
What is it? Create array in order to create array?

Reply:
Addition input validation is required.

```python
import math
from typing import Sequence, TypeAlias

import numpy as np

ArrayLike: TypeAlias = Tensor | np.ndarray | ...


def array(object, tiling: Sequence[int], dtype=None, ...) -> Tensor:
    match object:
        case np.ndarray():
            tmp = object.astype(dtype=dtype, order='F', copy=False)
            arr = empty_like(tmp, tiling=tiling)
            arr.from_array(tmp)
        case _:
            raise NotImplementedError(
                f'No coersion from {type(object)} to tiled array is '
                'implemented at the momment.')


def empty(shape: Sequence[int], tiling: Sequence[int],
          dtype=None) -> Tensor:
    dtype = dtype or np.float32
    match dtype:
        case np.float32:
            tensor_ty = Tensor_fp32
        case np.float64:
            tensor_ty = Tensor_fp64
        case _:
            raise NotImplementedError(
                f'Tensor does not support {dtype} dtype.')

    bounds = ()
    for dim, tile_dim in zip(shape, tiling):
        if dim % tile_dim != 0:
            raise NotImplementedError(
                'Tile shape does not fit tensor shape evenly.')
        bounds += (dim // tile_dim,)
    notiles = math.prod(bounds)

    # TODO(@daskol): Human-friendly interface for device placement.
    placement = [0] * notiles
    traits = TensorTraits(shape, tiling)
    return tensor_ty(traits, placement, 0)


def empty_like(arr: ArrayLike, tiling: Sequence[int] | None = None,
               dtype=None) -> Tensor:
    if tiling is None:
        match arr:
            case np.ndarray():
                tiling = arr.shape
            case Tensor():
                tiling = arr.tiling
            case _:
                raise TypeError(
                    f'Unexpected array-like value of {type(arr)} type.')
    return empty(arr.shape, tiling, arr.dtype)


def full(shape: Sequence[int], tiling: Sequence[int], fill_value, dtype=None):
    arr = empty(shape, tiling, dtype)
    fill_value = arr.dtype(fill_value)
    fill_async(fill_value, arr)
    return arr


def ones(shape: Sequence[int], tiling: Sequence[int], dtype=None):
    return full(shape, tiling, 1, dtype)


def zeros(shape: Sequence[int], tiling: Sequence[int], dtype=None):
    return full(shape, tiling, 0, dtype)
```

Reply:
thanks! Took your code as baseline